pandas/io/excel.py

```diff
@@ -804,6 +804,8 @@ class ExcelWriter(object):
     datetime_format : string, default None
         Format string for datetime objects written into Excel files
         (e.g. 'YYYY-MM-DD HH:MM:SS')
+    mode : {'w' or 'a'}, default 'w'
+        File mode to use (write or append).
```

Comment:
add a versionadded

Reply:
Do you think it's worth adding this class to the API? Would be a little strange as its an ABCMeta but on the flip side may still be beneficial for those looking further at it

Reply:
> Do you think it's worth adding this class to the API?

Can't you just do `from pandas.io.excel import ExcelWriter` ?

Reply:
It's not the import mechanism I was referring to as much as the documentation - this class is not part of the doc build

Reply:
@WillAyd : Ah, thanks for clarifying.  I don't see why not then, but let's see what @jreback has to say about that.